catalyst_voices/lib/pages/account/unlock_keychain_dialog.dart

```diff
@@ -0,0 +1,168 @@
+import 'package:catalyst_voices/common/error_handler.dart';
+import 'package:catalyst_voices/pages/registration/pictures/unlock_keychain_picture.dart';
+import 'package:catalyst_voices/pages/registration/widgets/information_panel.dart';
+import 'package:catalyst_voices/pages/registration/widgets/registration_stage_message.dart';
+import 'package:catalyst_voices/widgets/snackbar/voices_snackbar.dart';
+import 'package:catalyst_voices/widgets/snackbar/voices_snackbar_type.dart';
+import 'package:catalyst_voices/widgets/widgets.dart';
+import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:catalyst_voices_blocs/catalyst_voices_blocs.dart';
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:catalyst_voices_services/catalyst_voices_services.dart';
+import 'package:catalyst_voices_view_models/catalyst_voices_view_models.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+/// A dialog which allows to unlock the session (keychain).
+class UnlockKeychainDialog extends StatelessWidget {
+  const UnlockKeychainDialog({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return VoicesTwoPaneDialog(
+      left: InformationPanel(
+        title: context.l10n.unlockDialogHeader,
+        picture: const UnlockKeychainPicture(),
+      ),
+      right: const _UnlockPasswordPanel(),
+    );
+  }
+
+  static Future<void> show(BuildContext context) {
+    return VoicesDialog.show(
+      context: context,
+      routeSettings: const RouteSettings(name: '/unlock'),
+      builder: (context) => const UnlockKeychainDialog(),
+      barrierDismissible: false,
+    );
+  }
+}
+
+class _UnlockPasswordPanel extends StatefulWidget {
```

Comment:
I feel like this widget is not necessary. What do you think about making `UnlockKeychainDialog` a `StatefulWidget` and extracting InfoPanel into stateless `_UnlockInformationPanel`?

Reply:
Ok, moved the logic to `UnlockKeychainDialog` and converterd `_UnlockInformationPanel` to stateless